src/features/people/PeopleDetails/index.js

```diff
@@ -53,12 +52,21 @@ const PersonDetails = () => {
         return date.getFullYear();
     };
 
-    const formatVote = (vote) => vote.toLocaleString(
-        'pl-PL',
-        {
-            maximumFractionDigits: 1,
-        }
-    );
+    const formatVote = (vote) => {
+        const roundedNumber = vote.toFixed(1);
+        const localeString = (number) => number.toLocaleString(
+            'pl-PL',
+            {
+                maximumFractionDigits: 1,
+            }
+        );
+
+        if (Number.isInteger(vote) || Number.isInteger(+roundedNumber)) {
+            return `${localeString(vote)},0`
+        } else {
+            return localeString(vote)
+        };
+    };
 
     return (
         status === "loading" ? <div>Loading...</div>
```

Comment:
I think we should use components from common/Loading since we have them 🙂

Reply:
Thank you for pointing that out! 😊 I appreciate your suggestion. I did attempt to use the components from common/Loading, but encountered a parsing error in the process: 
```
ERROR in [eslint]
src\common\Loading\index.js
  Line 3:15:  Parsing error: Invalid left-hand side in assignment expression. (3:15)
```
Could you also please check if you have any insights into resolving the parsing error? Your input would be valuable in getting this sorted out smoothly. Thank you in advance 😊

Reply:
yes 🤦‍♂️

Reply:
could You change it in Loading/index to
```
export const Loading = () => (
    < Container >
        <SpinnerIcon />
    </Container >
);
```
and of course import in People Details inside braces { Loading }

Reply:
Already done 🙂